project_event/models/project_template.py

```diff
@@ -0,0 +1,20 @@
+# coding: utf-8 -*-
+# © 2018 Savoir-faire Linux
+# License LGPL-3.0 or later (http://www.gnu.org/licenses/lgpl).
+
+from odoo import fields, models
+
+
+class ProjectTemplate(models.Model):
+    _name = "project.template"
+    _inherit = ['project.project']
```

Comment:
@rimbendhaou I was wondering... do we really need to inherit from `projet.project`? If we are just fullfiling a couple of fields I guess we could simply not inherit the project.